tests/golden_html/codex-basic.html

```
@@ -0,0 +1,337 @@
+                  <th>words</th>
+                  <td>20</td>
+                </tr>
+                
+              </tbody>
+            </table>
+          </div>
+          <div class="content" id="content"><h2 id="user">User</h2>
+<p>Hello from Codex.</p>
+<h2 id="model">Model</h2>
+<p>Footnote: [9] ok.</p>
+<h2 id="model">Model</h2>
```

HTMLHint (1.9.2)

[error] The id value [ model ] must be unique.

(id-unique)